android/MLCChat/mlcengineexample/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
let us make mlcenginexample independent from the MLCChat, as in iOS case. we can use separate packaging list via mlc-package-config.json